manifests/install.pp

```diff
@@ -64,6 +67,7 @@
       }
 
       # Puppet Enterprise 3.8 and ships an embedded r10k so thats all thats supported
+      # Puppet FOSS 4.2 and up also ships an embedded r10k so thats all that should be supported
```

Comment:
Yeah.  I copied the comment line.  I meant to change r10k to ruby.  Will get that changed.